src/mtk_constructor.jl

```diff
@@ -253,16 +394,67 @@ function save_first_order_module(H; ȳ = nothing, x̄ = nothing, ȳ_iv = nothing
         verbose &&
             printstyled("Model already generated at $model_cache_path\n", color = :cyan)
     else
-        mod = generate_first_order_model(H; y, x, ȳ, x̄, ȳ_iv, x̄_iv, Γ, Ω, η, Q, p, p_f,
-                                         functions_type, simplify, parallel, skipzeros,
-                                         verbose)
-        @unpack n, n_y, n_x, n_p, n_ϵ, n_z, η, Q, Γ_expr, Γ_p_expr, Ω_expr, Ω_p_expr, H_expr, H_yp_expr, H_y_expr, H_xp_expr, H_x_expr, H_yp_p_expr, H_y_p_expr, H_xp_p_expr, H_x_p_expr, H_p_expr, Ψ_expr, H̄_expr, H̄_w_expr, ȳ_iv_expr, x̄_iv_expr, ȳ_expr, x̄_expr, ȳ_p_expr, x̄_p_expr, functions_type, allocate_solver_cache_expr = mod
+        mod = generate_first_order_model(
+            H;
+            y,
+            x,
+            ȳ,
+            x̄,
+            ȳ_iv,
+            x̄_iv,
+            Γ,
+            Ω,
+            η,
+            Q,
+            p,
+            p_f,
+            functions_type,
+            simplify,
+            parallel,
+            skipzeros,
+            verbose,
+        )
+        @unpack n,
+        n_y,
+        n_x,
+        n_p,
+        n_ϵ,
+        n_z,
+        η,
+        Q,
+        Γ_expr,
+        Γ_p_expr,
+        Ω_expr,
+        Ω_p_expr,
+        H_expr,
+        H_yp_expr,
+        H_y_expr,
+        H_xp_expr,
+        H_x_expr,
+        H_yp_p_expr,
+        H_y_p_expr,
+        H_xp_p_expr,
+        H_x_p_expr,
+        H_p_expr,
+        Ψ_expr,
+        H̄_expr,
+        H̄_w_expr,
+        ȳ_iv_expr,
+        x̄_iv_expr,
+        ȳ_expr,
+        x̄_expr,
+        ȳ_p_expr,
+        x̄_p_expr,
+        functions_type,
+        allocate_solver_cache_expr = mod
 
         mkpath(model_cache_location)
         open(model_cache_path, "w") do io
             write(io, "module $(model_name)\n")
-            write(io,
-                  "using SparseArrays, LinearAlgebra, DifferentiableStateSpaceModels, LaTeXStrings, ModelingToolkit\n")
+            write(
+                io,
+                "using SparseArrays, LinearAlgebra, DifferentiableStateSpaceModels, LaTeXStrings, ModelingToolkit\n",
+            )
```

Comment:
This was the one line I wasn't sure about. What did we need `LaTeXStrings` for? @wupeifan or maybe @jlperla might know.